examples/nlp/text_normalization_as_tagging/dataset_preparation/prepare_corpora_for_alignment.py

```diff
@@ -47,7 +47,7 @@
 from os.path import isdir, join
 from shutil import rmtree
 
-from examples.nlp.text_normalization_as_tagging.dataset_preparation.utils import get_src_and_dst_for_alignment
+from nemo.collections.nlp.data.text_normalization_as_tagging.utils import get_src_and_dst_for_alignment
```

Comment:
add file with requirement on how to install giza

Reply:
done